igvm/commands.py

```diff
@@ -51,6 +51,42 @@ def _check_defined(vm, fail_hard=True):
             log.info(error)
 
 
+@with_fabric_settings
+def evacuate(hv_hostname, offline=None, dry_run=False):
+    """Move all VMs out of a hypervisor
+
+    Move all VMs out of a hypervisor and put it to state maintenance.
+
+    Offline can be passed without arguments or with a list strings matching
+    function attributes. If just passed all VMs will be migrated offline. If
+    a list of strings is passed only those matching will be migrate offline.
+    """
+
+    with _get_hypervisor(hv_hostname, ignore_reserved=True) as hv:
+        if dry_run:
+            log.info('Setting {} to state maintenance'.format(hv_hostname))
```

Comment:
The message in dry-run should be the opposite, saying "not setting" or "I would set". Also there is no message at all when no dry-run happens. This relates to all messages below.